src/next/handler.ts

```diff
@@ -0,0 +1,14 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { serialize } from "cookie";
+import { SaleorExternalAuth } from "../SaleorExternalAuth";
+
+export const createSaleorExternalAuthHandler =
```

Comment:
We should update `README.md` an add a simple example on how to use `createSaleorExternalAuthHandler`